src/StepDefinition/MinkElementTrait.php

```diff
@@ -185,4 +185,63 @@ public function textAppearsTimesInElement($text, $element, $number, $selector =
     }
   }
 
+  /**
+   * @Then /^(?:|I )wait (?P<waitTime>\d+) ms for (?P<num>\d+) "(?P<element>[^"]*)" elements to appear?$/
+   * @Then /^(?:|I )wait for (?P<num>\d+) "(?P<element>[^"]*)" elements to appear?$/
+   *
+   * @param string $element
+   * @param int $num
+   * @param int $waitTime
+   *
+   * @throws \Exception
+   */
+  public function iWaitForElementToAppear($waitTime = 5000, $num, $element) {
+    $this->spinWaitForElementsToAppear($waitTime, function ($context) use ($element, $num) {
+      try {
+        $this->assertSession()->elementsCount('css', $element, intval($num));
+        return TRUE;
+      }
+      catch (ExpectationException $e) {
+
+      }
+      return FALSE;
+    });
+  }
+
+  /**
+   * Implementation of spin method to wait $waitTime for
+   * elements to appear.
+   *
+   * @param string $lambda
+   *   Function expresion.
+   *
+   * @return bool
+   *
+   * @throws \Exception
+   */
+  private function spinWaitForElementsToAppear($waitTime, $lambda) {
```

Comment:
https://github.com/minkphp/Mink/issues/482 - isn't that waitFor() with callback pretty much that? Can you use that instead?